### PR TITLE
Refactor cucumber scenarios

### PR DESCRIPTION
**Pull request by Paola-Xaq**

### CODE REVIEW

File: src/test/resources/features/PriceBook.feature [hunk 1]
```diff
@@ -1,122 +1,155 @@
...
     When I execute create "PRICE_BOOK" request
-      | name | Created PriceBook |
-    Then Status response of request should be "CREATED"
-
-  @CreateAPriceBook
-  Scenario: Create a priceBook with one character
+      | name | Price name book |
+    Then "response" schema status response of request should be "CREATED"
+
+  #@UseCreatedContact
```
Comment (edsonarios) [inline src/test/resources/features/PriceBook.feature]: This line comments should be erase

File: src/test/resources/features/contact.feature [hunk 2]
```diff
@@ -1,48 +1,70 @@
...
+  @DeleteContact
+  Scenario: Validate Schema of a Contact
     Given I build "POST" request
     When I execute create "CONTACT" request
-      | lastname | "#$%/ New lastname #$%/" |
-    Then Status response of request should be "CREATED"
+      | lastname | Ramirez Perez |
+    Then "response" schema status response of request should be "CREATED"
+
+  #@UseCreatedContact
```
Comment (edsonarios) [inline src/test/resources/features/contact.feature]: This line comments should be erase

File: src/test/resources/features/contact.feature [hunk 3]
```diff
@@ -51,82 +73,115 @@ Feature: Scenario test for Contact feature
...
-    When I execute update "CONTACT_ID" request with specific id "  5005e000000xr4MAAQ"
-      | lastname | " " |
-    Then Status response of request should be "NOT_FOUND"
\ No newline at end of file
+    Examples:
+      | id                    |
+      | " Mv234498cvXvmsj435" |
+      | "Mv234498cvXvmsj435"  |
+
+  #@UseCreatedCampaign
```
Comment (edsonarios) [inline src/test/resources/features/contact.feature]: This line comments should be erase

**Review by JorgeRCV — APPROVED** [review]
Well done 🥇

**Review by edsonarios — APPROVED** [review]

**Pull request merged**